src/components/ModularScreen/ModularScreen.tsx

```diff
@@ -65,14 +67,17 @@ export function ModularRightOptions<
       onValueChange(options[newIndex].id);
     }
   };
-  useHandleGestures({
-    left: () => {
-      handleTurn(-1);
+  useHandleGestures(
+    {
+      left: () => {
+        handleTurn(-1);
+      },
+      right: () => {
+        handleTurn(1);
+      }
     },
-    right: () => {
-      handleTurn(1);
-    }
-  });
+    shouldIgnoreGesture
```

Comment:
Wouldn't it make sense to have a selection for the bubble display here as well to ensure a re-render and an update of that hook regardless of calling order?